lib/client.js

```diff
@@ -21,7 +21,9 @@ var Client = function(options) {
 
   options = options || {};
 
-  log4js.setGlobalLogLevel(log4js.levels[options.logLevel || 'WARN']);
+  var logger = log4js.getLogger();
+  logger.level = 'warn';
+  logger.debug("Some debug messages");
```

Comment:
Should this line really be here?

Reply:
Actually, "log4js.getLogger().level = 'warn'" would go just fine, that was just hot-fix from log4js repository. Btw, debug messages wouldn't show in 'warn' level logger set up.

Reply:
Would prefer that 👍